### DISCUSSION TITLE
Some pages are missing or not found

### DISCUSSION THREAD

**Discussion by SPyofgame** [facelessuser/pymdown-extensions#1998] [Q&A] [answered]
Like in this [about/development](https://facelessuser.github.io/pymdown-extensions/about/development/)

![image](https://user-images.githubusercontent.com/62207095/227569094-2e91903a-eb1c-48ce-9067-4f294e65a5af.png)

Maybe links that no longer work can have a little icon next to it for indication of a broken link ?

![image](https://user-images.githubusercontent.com/62207095/227569370-59875ad1-029d-4604-b75c-7e076cc051f8.png)

**Comment by facelessuser** [comment, marked as answer]
Yeah, I should update the links. There is no validation of the links, so changing the icons is not possible.
